src/main/java/sirius/biz/jupiter/JupiterSync.java

```diff
@@ -100,6 +100,10 @@
     @ConfigValue("jupiter.repository.uplink.bucket")
     private String uplinkBucket;
 
+    @ConfigValue("jupiter.repository.uplink.test-bucket")
```

Comment:
Do we really need this config? Why not just set the previous param to another bucket in dev/test/staging?